### PR TITLE
docs: improve eslint loader msg + add cookbook example

### PR DESCRIPTION
**Pull request by gioboa**
# Overview

# What is it?

- [ ] Feature / enhancement
- [ ] Bug
- [ ] Docs / tests / types / typos

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

# Use cases and why

- 1. One use case
- 2. Another use case

# Checklist:

- [ ] My code follows the [developer guidelines of this project](https://github.com/BuilderIO/qwik/blob/main/CONTRIBUTING.md)
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

File: packages/docs/src/routes/docs/cookbook/re-exporting-loaders/index.mdx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+title: Cookbook | Re-exporting loaders
+contributors:
+  - gioboa
+updated_at: '2023-12-15T11:00:00Z'
+created_at: '2023-12-15T11:00:00Z'
+---
+
+# Re-exporting loaders
+
+`routeAction$` and `routeLoader$` are typically declared in route boundary files such as layout.tsx, index.tsx and plugin.tsx inside the routesDir directory
```
Comment (mhevery) [inline packages/docs/src/routes/docs/cookbook/re-exporting-loaders/index.mdx:11]: I think there are two examples:

1. My `indext.tsx` file is too pig and I want to pull code in a separate file as shown in this example.
2. I am writing third-party lib and just want to add an existing component. Now i need to re-export the component router functions. 

I don't think the second case is covered.

File: packages/docs/src/routes/docs/cookbook/re-exporting-loaders/index.mdx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+// eslint-disable-next-line qwik/loader-location
+export const useCommonRouteAction = routeAction$(async () => {
+  // ...
+});
+
+// eslint-disable-next-line qwik/loader-location
+export const useCommonRouteLoader = routeLoader$(async () => {
+  // ...
+});
+```
```
Comment (mhevery) [inline packages/docs/src/routes/docs/cookbook/re-exporting-loaders/index.mdx:36]: I am always concerned tha these code examples will go out of date ar will have syntax error. Can we turn them into real code? See any `MDX` file which contains

```
import CodeSandbox, {CodeFile} from '../../../../components/code-sandbox/index.tsx';
```

File: packages/docs/src/routes/docs/cookbook/re-exporting-loaders/index.mdx (hunk 1 above)
Comment (gioboa) [reply packages/docs/src/routes/docs/cookbook/re-exporting-loaders/index.mdx:13]: ✅ done

File: packages/docs/src/routes/docs/cookbook/re-exporting-loaders/index.mdx (hunk 2 above)
Comment (gioboa) [reply packages/docs/src/routes/docs/cookbook/re-exporting-loaders/index.mdx:46]: ✅ done

**Pull request merged**